### PR TITLE
Full pt-br translation

### PR DESCRIPTION
**Pull request by Ednaxx**
Finished pt-br translation. On assets related to other mods, we tried using their proper translation as used on such mods (when possible).

### CODE REVIEW

File: common/src/main/resources/assets/adorn/lang/pt-br.json
Comment (Juuxel) [inline common/src/main/resources/assets/adorn/lang/pt-br.json:1]: I think the file should be called `pt_br.json` with an underscore.

**Review by Juuxel — CHANGES_REQUESTED** [review]
Looking at the Brazilian Portuguese translation of Minecraft itself, names of blocks/items that have multiple words only have the first word capitalised. Adorn's translations should follow that rule too to match vanilla.

**Comment by Ednaxx** [comment]
Sure! I've just fixed what you pointed.

**Review by Juuxel — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**